lib/utils.js

```diff
@@ -161,14 +161,14 @@ function buildCustomNotification(_) {
 
     // Properties with simple API variants
     NotificationTypeKey: getNotificationTypesKeyWithPrefix(_.NotificationTypeKey || _.type),
-    Recipients: _.Recipients || _.recipients?.map((recipient) => ({ RecipientId: recipient })),
+    Recipients: _.Recipients || _.recipients?.map(id => ({ RecipientId: id })),
     Priority: _.Priority || _.priority || "NEUTRAL",
-    Id: _.Id || _.id,
-    Properties: _.Properties || _.properties || Object.entries(_.data).map(([k,v]) => ({
+    Properties: _.Properties || Object.entries(_.data).map(([k,v]) => ({
       Key:k, Value:v, Language: "en", Type: typeof v, // IsSensitive: false
     })),
 
     // Low-level API properties
+    Id: _.Id,
```

Comment:
as @simeonPetkov96 said we can remove Id because we do not encourage customers to use it